working-groups/grammar/README.md

```diff
@@ -0,0 +1,34 @@
+# Grammar Working Group
+![working group status: active][status]
+
+The goal of the working group is to satisfy [RFC 1331] and produce a testable,
+canonical grammar for the Rust language. The primary audiences for the grammar
+are:
+
+- Rust RFC authors who wish to propose and communicate changes.
+- rustc and Rust tool developers who need an authoritative definition of the
+  grammar.
+- To assist documentation efforts to communicate valid Rust syntax to users,
+  and to facilitate the [Rust language specification].
+
+- **Leads:** [@centril][centril], [@qmx][qmx]
+- **Meeting Notes:** [All][meeting-notes]
+- **Dedicated repository:** [rust-lang-nursery/wg-grammar][wg-grammar]
+
+[status]: https://img.shields.io/badge/status-active-brightgreen.svg?style=for-the-badge
+[centril]: https://github.com/centril
+[qmx]: https://github.com/qmx
+[wg-grammar]: https://github.com/rust-lang-nursery/wg-grammar
+[meeting-notes]: https://github.com/rust-lang-nursery/wg-grammar/blob/master/misc/meeting-notes.md
+
+# How can I get involved?
+
+Meetings take place on the [#wg-grammar channel on Discord][discord] at [20:00
```

Comment:
Can you add this to the lang team calendar? You should have access now

Reply:
And include a link to the calendar section of the main README

Reply:
done, also updated to a copy that will require less maintenance 😃